suggestionBox/suggestionBox-library/src/main/java/org/silverpeas/components/suggestionbox/model/SuggestionCollection.java

```diff
@@ -122,7 +121,7 @@ public void add(final Suggestion suggestion, final Collection<UploadedFile> uplo
   @Override
   public boolean remove(Object aSuggestion) {
     final Suggestion suggestion = (Suggestion) aSuggestion;
-    boolean r = Transaction.performInOne(new Transaction.Process<Boolean>() {
+    boolean r = Transaction.performInOne(new Process<Boolean>() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1604)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "r". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1488)